services/src/test/java/org/keycloak/test/broker/oidc/AbstractOAuth2IdentityProviderTest.java

```diff
@@ -121,9 +137,54 @@ public void getFederatedIdentity_responseUrlLine() {
 	}
 
 	private TestProvider getTested() {		
-		return new TestProvider(getConfig(null, null, null, Boolean.FALSE));
+		return new TestProvider(getConfig("https://localhost:8080/callback", null, "clientId", Boolean.FALSE));
 	}
 
+    @Test
```

Comment:
My vote is to rather remove all the tests you added in AbstractOAuth2IdentityProviderTest. You are already testing this feature more properly in the integration test KcOidcBrokerParameterForwardTest and it's no need to duplicate stuff and create another place for unecessary maintenance.